tests/data/ingestd.yml

```diff
@@ -6,16 +6,16 @@ num_messages: 50
 butler:
     repo: /tmp/repo
 
-rses:
-    XRD1:
+topics:
```

Comment:
I don't know how difficult it is, but it might be nice to test same-RSE, different scope (e.g. topic = `XRD1-test2`) even though we don't plan to use that mode.

Reply:
I added additional checks in two unit tests to do this.